### DISCUSSION TITLE
enh: foocus image generation engine support

### DISCUSSION THREAD

**Discussion by tuantupharma** [open-webui/open-webui#14291] [Ideas]
I hope open-webui support Fooocus in mage Generation Engine. It light weight and the image prompt is sometime better than comfyui, automatic1111.

**Comment by spammenotinoz** [comment]
FYI: If it helps in the current version, I use the models feature to create a custom model called "Stable Diffusion" with a system prompt.
"You are an expert in providing a short and concise image description for Stable Diffusion.  Provide no other explanation or reasoning, just a single descriptive prompt suitable for image generation."
Not claiming it's the best system prompt, but is working well.  For the model I am using a rather uncensored model from openrouter.  "openrouter/neversleep/llama-3-lumimaid-8b"
You also have the EDIT icon, and clicking the "Generate Image" icon again and again creates more images.
What may work well is a dummy model called "IMAGE" that doesn't use any model, just sends prompts directly to the back-end image api.

**Comment by ItsNoted** [comment]
Came here to see if Fooocus could be supported. This project has been fantastic for image generation and would be awesome if we could implement this into open webui.

**Comment by jmtemmos** [comment]
Hello. It would be great to have Fooocus support :) agreed :)

**Comment by dot-Justin** [comment]
@tjbck I see you've self-assigned this, this repo may be useful: https://github.com/mrhan1993/Fooocus-API
